### ISSUE TITLE
[Feature] Security Exposure Management module

### ISSUE THREAD

**Issue by piaudonn** [briandelmsft/SentinelAutomationModules#453]
Query Security Exposure Management data to return blast radius, number of paths going from or to a user or device as well as the highest vulnerability level of the nodes. Can use the make-graph operators on the exposure nodes and edges table. 
Maybe support azure resource as an entity too.

**Comment by piaudonn** [comment]
This should probably replace few of the feature we currently offer in the MDE module. We could use it to list devices used by a user and its associated level of risk. Maybe the MDE should merge into an XDR where we would also do MDI specific things (like on-prem AD things). MDE, MDCA (retired), MDI (well AD on-prem stuff) converging to an XDR one. Also using the XDR endpoint of the Graph API for KQL queries. @briandelmsft what do you think?

**Comment by briandelmsft** [comment]
@piaudonn sounds good, I think we'll need to put together a plan for what each module will do and how we will get that data before we can really understand the scope of the work.  We may also need to start looking at async calls for some of this data collection as we currently already run the risk of the 2 minutes before the connector time outs, and adding more into a single module could amplify that risk
